src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,40 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String CONSTANTECHAR = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder builder = new StringBuilder();
+        try {
+            Path paths = Paths.get(fileName);
+            File file = paths.toFile();
+            InputStream inputStream = new FileInputStream(file);
+
+            while (inputStream.available() > 0) {
+                char tmp = (char) inputStream.read();
+                builder.append(tmp);
+            }
+            inputStream.close();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+        String[] words = builder.toString().toLowerCase().split(" ");
```

Comment:
do words on different lines get separated in this case?
what should we use in `split()` to make this work?